hypertrace-view-generator/hypertrace-view-generator/src/main/resources/configs/view-gen-log-event/application.conf

```diff
@@ -0,0 +1,39 @@
+service.name = view-gen-log-event
+service.admin.port = 8099
+
+main.class = org.hypertrace.core.viewgenerator.service.ViewGeneratorLauncher
+
+input.topic = "raw-logs"
+output.topic = "log-event-view"
+input.class = org.hypertrace.core.datamodel.LogEventRecords
```

Comment:
this will be just `LogEvents` right? 
https://github.com/hypertrace/data-model/blob/main/data-model/src/main/avro/LogEvent.avdl